modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerExtension.java

```diff
@@ -71,6 +81,7 @@ public void setVersion(String version) {
 	private Object _baseDir = "poshi-tests";
 	private String _openCVVersion = "2.4.9-0.9";
 	private final Map<String, Object> _poshiProperties = new HashMap<>();
+	private Object _poshiRunnerProperties = "poshi-runner.properties";
```

Comment:
I talked to @lawrence-lee, I would like to let people choose where they want to have their properties file and how to call it. I checked Poshi's source code, it seems it can read a `poshi-runner-ext.properties` file from its classpath, so maybe we can try to:

1. let people set the Poshi runner properties file location in the `poshiRunner.poshiPropertiesFile` extension property
2. when it's time to add dependencies to the `poshiRunner` configuration, we check if that file exists
3. if it does, we copy it to a temp dir while renaming it to `poshi-runner-ext.properties` and add it to the configuration, so hopefully Poshi will read it